README.md

```diff
@@ -61,7 +61,7 @@ Package [samza.examples.cookbook](https://github.com/apache/samza-hello-samza/tr
 Package [samza.examples.wikipedia.application](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/wikipedia/application) contains a small Samza application which consumes the real-time feeds from Wikipedia, extracts the metadata of the events, and calculates statistics of all edits in a 10-second window. You can start the app on the grid using the run-app.sh script:
 
 ```
-./deploy/samza/bin/run-app.sh --config-factory=org.apache.samza.config.factories.PropertiesConfigFactory --config-path=file://$PWD/deploy/samza/config/wikipedia-application.properties
+./deploy/samza/bin/run-app.sh --config-path=$PWD/deploy/samza/config/wikipedia-application.properties
```

Comment:
This `--config-path` here is for submission configs only, right? Can you think of a way to help clarify that? It looks like the example is suggesting to pass full job configs as submission configs. However, that could end up being a problem if the full job configs are too large (IIRC, there is a limit for the size of the submission configs env variable when passing to YARN).

Reply:
On the other hand, if someone switched this to use `LocalApplicationRunner`, then would the `--config-path` need to be the full configs? Would this overloading of the `--config-path` argument be confusing?

Reply:
`LocalApplicationRunner` with `--config-path` is still submission config, no full config is needed.